include/core/evaporation_model.h

```diff
@@ -0,0 +1,683 @@
+/* ---------------------------------------------------------------------
+ *
+ * Copyright (C) 2019 -  by the Lethe authors
+ *
+ * This file is part of the Lethe library
+ *
+ * The Lethe library is free software; you can use it, redistribute
+ * it, and/or modify it under the terms of the GNU Lesser General
+ * Public License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ * The full text of the license can be found in the file LICENSE at
+ * the top level of the Lethe distribution.
+ *
+ * ---------------------------------------------------------------------
+ */
+#ifndef lethe_evaporation_model_h
+#define lethe_evaporation_model_h
+
+#include <core/parameters.h>
+// only for the field definition
+#include <core/physical_property_model.h>
+
+using namespace dealii;
+
+/**
+ * @brief EvaporationModel. Abstract class that allows to calculate the
+ * evaporation mass, heat and momentum fluxes.
+ */
+class EvaporationModel
+{
+public:
+  EvaporationModel()
+  {
+    model_depends_on[temperature] = false;
```

Comment:
If the model does not depend on temperature, don't we by default assume all models don't depend on anything? Is it necessary to respecify this information here?